packages/sx05re/emulators/ecwolf/scripts/ecwolf.sh

```diff
@@ -5,6 +5,8 @@
 
 . /etc/profile
 
+# We cd into this directory before starting ecwolf
+RUN_DIR="/emuelec/configs/ecwolf"
```

Comment:
Why not use $CONFIG_DIR? its the same path

Reply:
Oh you overwrite it later on, I get it

Reply:
You mean
RUN_DIR="${CONFIG_DIR}"

Reply:
Yes but I was mistaken